docs/information_architecture.rst

```diff
@@ -115,17 +115,15 @@ default commerce entities generated by this module get deleted upon installation
 
 :Order Item type:
 
-    There are 2 types of order item types depending on the submitted donation type:
-
-    - **Single Donation** [single_donation] - Defines Order Item type for single donation.
-    - **Recurring Donation** [recurring_donation] - Defines Order Item type for recurring donation.
+    There is single order item type **Donation** [donation].
```

Comment:
Had to change it here - we can't have different order item types added to the same order type.